src/mpc/structs.jl

```diff
@@ -321,6 +503,43 @@ Base.@kwdef struct MPCCoolingLoad
     cop::Union{Real, Nothing}
 end
 
+"""
+    MPCDomesticHotWaterLoad
+"""
+Base.@kwdef struct MPCDomesticHotWaterLoad
+    loads_kw_thermal::Array{Real,1}
+end
+
+"""
+    MPCSpaceHeatingLoad
+"""
+Base.@kwdef struct MPCSpaceHeatingLoad
+    loads_kw_thermal::Array{Real,1}
+end
+
+
+"""
+    MPCHighTempThermalStorage
+"""
+Base.@kwdef struct MPCHighTempThermalStorage <: AbstractThermalStorage
+    charge_limit_kw::Float64
+    discharge_limit_kw::Float64
+    size_kwh::Float64
+    charge_efficiency::Float64 = 1.0
+    discharge_efficiency::Float64 = 0.9
+    soc_min_fraction::Float64 = 0.2
+    soc_init_fraction::Float64 = 0.5
+    can_grid_charge::Bool = true
+    grid_charge_efficiency::Float64 = can_grid_charge ? charge_efficiency : 0.0
+    size_kw::Float64 = charge_limit_kw + discharge_limit_kw
```

Comment:
why would the size be this sum? do we even want this field? looks like the constraint that uses it is commented out and would be duplicative since we already have the charge and discharge limits.